server/android/src/main/java/dev/slimevr/android/tracking/trackers/hid/TrackersHID.kt

```diff
@@ -0,0 +1,292 @@
+package dev.slimevr.android.tracking.trackers.hid
+
+import android.app.PendingIntent
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.hardware.usb.UsbDevice
+import android.hardware.usb.UsbManager
+import androidx.core.content.ContextCompat
+import dev.slimevr.VRServer
+import dev.slimevr.tracking.trackers.Device
+import dev.slimevr.tracking.trackers.Tracker
+import dev.slimevr.tracking.trackers.TrackerStatus
+import dev.slimevr.tracking.trackers.hid.HIDCommon
+import dev.slimevr.tracking.trackers.hid.HIDCommon.Companion.HID_TRACKER_RECEIVER_PID
+import dev.slimevr.tracking.trackers.hid.HIDCommon.Companion.HID_TRACKER_RECEIVER_VID
+import dev.slimevr.tracking.trackers.hid.HIDCommon.Companion.PACKET_SIZE
+import dev.slimevr.tracking.trackers.hid.HIDDevice
+import io.eiren.util.logging.LogManager
+import java.nio.ByteBuffer
+import java.util.function.Consumer
+
+const val ACTION_USB_PERMISSION = "dev.slimevr.USB_PERMISSION"
+
+/**
+ * Receives trackers data by UDP using extended owoTrack protocol.
+ */
+class TrackersHID(
```

Comment:
Maybe rename it to something like "AndroidHIDService", currently the name is not matching the purpose and is confusing.